x/bank/handler.go

```diff
@@ -30,14 +30,6 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 
 // Handle MsgSend.
 func handleMsgSend(ctx sdk.Context, k keeper.Keeper, msg *types.MsgSend) (*sdk.Result, error) {
-	if err := k.SendEnabledCoins(ctx, msg.Amount...); err != nil {
```

Comment:
I assume these had to do with vesting accounts? Is this logic moved elsewhere now or is it irrelevant?

Reply:
nothing to do with vesting.

Reply:
It was moved to x/bank `SendCoins`